exp/sampler.jl

```diff
@@ -114,8 +114,14 @@ if abspath(PROGRAM_FILE) == @__FILE__
     D["input"] = Dict{String,Any}(
         "meta" => Dict{String,Any}("seed" => Int[]),
         "data" => Dict{String,Any}(
+            "reserve_requirement" => Float64[],
```

Comment:
```suggestion
            "minimum_reserve" => Float64[],
```
All other keys match the field name of OPFData